helpers/filterSeats.js

```diff
@@ -86,8 +111,56 @@ module.exports = function(dust) {
       });
     }
 
+    function reorderRates(rates, replyIndex) {
+      var bandTypes = {
+        'gold': [],
+        'silver': [],
+        'bronze': []
+      };
```

Comment:
Can we get these bandTypes from those used in Transformer, in case some change/get added in future?